### ISSUE TITLE
Create a Readme and a landing page

### ISSUE THREAD

**Issue by LeanderK** [intellimate/Izou#18]
Nobody understands what we are doing since we are simply not explaining it.

We must create a ReadMe and a landing page would be neat.
I propose a short & concise ReadMe like: https://github.com/scalaz/scalaz
And a simple landing page where we can write a short tutorial and explain some concepts.

When this proposal is finished and well executed we can promote our project. (Or wait for proposal: https://github.com/intellimate/Izou/issues/16 - this would make it possible to contribute).

The goal of this proposal is:
-create a readme
-create a homepage
-make people understand what we are doing

**Comment by jundl77** [comment]
I agree something we definitely need. I think the proposed readme is good, but I would, as an introduction, get a bit creative and write a short and concise story (fiction) explaining what izou is in order to spark interest. I realize this is a very difficult thing to do well, but I think for our case it can be huge if it is done well, just because it is very hard to picture what izou really is. And bombarding the reader with information and examples to illustrate izou's full potential does not seem as the best option to me. This has the advantage that we pop out from other github projects.

**Comment by LeanderK** [comment]
I don't think so. The most popular github projects are not the ones with the most creative Readme. This advertisement-like thing can maybe go to the website, but I wouldn't overdo it.

**Comment by jundl77** [comment]
Well most Github projects are also not so difficult to explain as ours. When you have a straight forward how-to kind of project, that might be a good idea, but for our case I think it it is not the best option. We can do the average and be average, or we can try to stand out, which we should try.

**Comment by jundl77** [comment]
Here let me give it a shot, if it doesn't work out we can always go the straight forward way.

**Comment by LeanderK** [comment]
okay
